### ISSUE TITLE
The mobile skip link to jump to the search results cannot be operated by assistive devices as the focusable element has been hidden.

### ISSUE THREAD

**Issue by christinach** [pulibrary/pul_library_drupal#2001]
The mobile skip link to jump to the search results cannot be operated by assistive devices as the focusable element has been hidden.

**Suggestion:**
This visible element is a <label> functioning as a bookmark link.
Probably adjusting the markup to the correct semantic element is the easiest fix. Exposing the input would be confusing, role-wise.

WCAG Guidelines:
https://www.w3.org/WAI/WCAG21/quickref/#bypass-blocks
